scrapers/ManyVids.yml

```diff
@@ -19,4 +29,69 @@ xPathScrapers:
 
-# Last Updated May 15, 2020
+  performerSearch:
+    performer:
+      Name: //*[contains(concat(' ', normalize-space(@class), ' '), ' profile-pic-name ')]/a/text()
+      URL:
+        selector: //*[contains(concat(' ', normalize-space(@class), ' '), ' profile-pic-name ')]/a/@href
+        replace:
+          - regex: '/(?:Store/Videos/)?$'
+            with: /About/
+
+  performerScraper:
+    common:
+      $aboutlabel: //*[@class='mv-about__container__details__list-unit']/span[@class='mv-about__container__details__list-label']
+    performer:
+      Name: //*[@class='mv-about__banner-name ']/text()
+      Gender:
+        selector: $aboutlabel[contains(text(), 'Gender')]/following-sibling::*
+        replace:
+          - regex: Trans
+            with: transgender_female
+      URL: //a[@data-tab='about']/@href
+      Twitter: //*[@class='mv-about__banner-social']/a/i[@class='icon twitter-icon']/../@href
+      Instagram: //*[@class='mv-about__banner-social']/a/i[@class='icon instagram-icon']/../@href
+      Birthdate:
+        selector: $aboutlabel[contains(text(), 'Age')]/following-sibling::*
+        replace:
+          - regex: ' \(.*\)$'
+            with: ''
+        parseDate: 2006.01.02
+      Ethnicity:
+        selector: $aboutlabel[contains(text(), 'Ethnicity')]/following-sibling::*
+      Country:
+        selector: $aboutlabel[contains(text(), 'Nationality')]/following-sibling::*
+      EyeColor:
+        selector: $aboutlabel[contains(text(), 'Eye Color')]/following-sibling::*
+      Height:
+        selector: $aboutlabel[contains(text(), 'Height')]/following-sibling::*
+        replace:
+          - regex: '.*?or '
+            with: ''
+          - regex: ' cm$'
+            with: ''
+          - regex: \D+[\s\S]+
+            with: ""
+      Measurements:
+        selector: $aboutlabel[contains(text(), 'Measurements')]/following-sibling::* | $aboutlabel[contains(text(), 'Breast Size')]/following-sibling::*
+      FakeTits:
+        selector: $aboutlabel[contains(text(), 'Breast Size')]/following-sibling::*[contains(text(), 'Cohesive Gel') or contains(text(), 'Silicon') or contains(text(), 'Saline')]
+        replace:
+          - regex: '\s+?[0-9]+.*'
+            with: ''
+      CareerLength:
+        selector: //*[@class='mv-about__banner-info']/strong/text()
+        replace:
+          - regex: '^Joined '
+            with: ''
+          - regex: $
+            with: ' - today'
+      Tattoos:
+        selector: $aboutlabel[contains(text(), 'Tattoos')]/following-sibling::*
+      Piercings:
+        selector: $aboutlabel[contains(text(), 'Piercings')]/following-sibling::*
+
+      Image:
+        selector: //*[contains(concat(' ',normalize-space(@class),' '),' mv-model-display ')][not(@data-current-portrait = '')]/@data-current-portrait | //*[contains(concat(' ',normalize-space(@class),' '),' mv-model-display ')][not(@data-current-avatar = '')]/@data-current-avatar
```

Comment:
is there a case where 
`selector: $pimg[not(@data-current-portrait = '')]/@data-current-portrait|$pimg[not(@data-current-avatar = '')]/@data-current-avatar`
doesn't work ?
where $pimg `$pimg: //div[@class="mv-model-display mv-model-display__block js-profile-display-container"]`
I only tried a few where it seems to work.

Reply:
Not every MV star has a portrait, but everyone has an avatar.
The default avatar is a data url containing an svg logo.
Not sure if that should be filtered out

Reply:
Imo it's ok either way default or none at all.